### PR TITLE
docs(Embeds): Added descriptions to the typedefs

### PR DESCRIPTION
**Pull request by Sardonyx78**
**Please describe the changes this PR makes and why it should be merged:**

**Status**

- [x] Code changes have been tested against the Discord API, or there are no code changes
- [ ] I know how to update typings and have done so, or typings don't need updating

**Semantic versioning classification:**

- [ ] This PR changes the library's interface (methods or parameters added)
  - [ ] This PR includes breaking changes (methods removed or renamed, parameters moved or removed)
- [x] This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

File: src/structures/MessageEmbed.js [hunk 1]
```diff
@@ -78,6 +79,7 @@ class MessageEmbed {
     }
 
     /**
+     * Represents thumbnail of a MessageEmbed
```
Comment (RDambrosio016) [inline src/structures/MessageEmbed.js:82]:
```suggestion
     * Represents the thumbnail of a MessageEmbed
```
This should be consistent with the other definitions below.

File: src/structures/MessageEmbed.js [hunk 2]
```diff
@@ -180,6 +186,7 @@ class MessageEmbed {
       : null;
 
     /**
+     * Represents the footer field of a MessageEmbed
```
Comment (RDambrosio016) [inline src/structures/MessageEmbed.js:189]:
```suggestion
     * Represents the footer of a MessageEmbed
```
Footer isnt really a field, it has no title, and the value doesnt support much formatting, it should be named independent of fields

Comment (RDambrosio016) [reply src/structures/MessageEmbed.js:189]: I think `author field` is a bit misleading too, but i cant think of a better name considering `the author of a MessageEmbed` is misleading too.

Comment (Sardonyx78) [reply src/structures/MessageEmbed.js:189]: Yeah its a tricky part.

File: src/structures/MessageEmbed.js [hunk 3]
```diff
@@ -62,6 +62,7 @@ class MessageEmbed {
     this.timestamp = data.timestamp ? new Date(data.timestamp).getTime() : null;
 
     /**
+     * Represents a field of a MessagEmbed
```
Comment (tipakA) [inline src/structures/MessageEmbed.js:65]: Typo

Comment (Sardonyx78) [reply src/structures/MessageEmbed.js]: Fixed

**Pull request merged**